apisix/plugins/limit-count/limit-count-redis-cluster.lua

```diff
@@ -37,6 +38,10 @@ local script = [=[
     return redis.call('incrby', KEYS[1], -1)
 ]=]
 
+--- compact the redis lua script
+script = string.gsub(script, "    ", "")
```

Comment:
Can we prepare a method/function to do this?

Reply:
Yes for sure, I'm wondering where it would be more appropriate to put it.

Reply:
Maybe `core.utils`?

Reply:
Better to put it in `core.string`. And the `ngx.re` version will be faster?

Reply:
Done

Reply:
@token ping